DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/SelfAssessmentDataService.cs

```diff
@@ -333,5 +337,32 @@ SELECT	s.DelegateUserID
             );
             return (delegateUserCard, ResultCount);
         }
+        public RemoveSelfAssessmentDelegate GetDelegateSelfAssessmentByCandidateAssessmentsId(int candidateAssessmentsId)
+        {
+            return connection.QueryFirstOrDefault<RemoveSelfAssessmentDelegate>(
+                 @"Select
+                    ca.Id AS CandidateAssessmentsId,
+                    ca.SelfAssessmentID,
+                    da.FirstName_deprecated AS FirstName, 

```

Comment:
the first name and last name columns are deprecated and should not be used. Please fetch the name from Users table

Reply:
Done